.github/workflows/nessie.yaml

```diff
@@ -492,3 +493,17 @@ jobs:
         # with:
         #   files: ./nessie-cover.out
         #   fail_ci_if_error: false
+
+  invoke-terraform-workflow:
+    name: Invoke Terraform Workflow to deploy Cryptolake ECS container
+    needs: [deploy-image, spark2, spark3, run-system-aws-s3, run-system-gcp-gs, run-system-azure-abfs]  
+    if: github.ref == 'refs/heads/cryptolake'
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Invoke Terraform Workflow
```

Comment:
Missing some information about that it is done using a `workflow_dispatch` event to the Treeverse's `terraform` repository.